sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,74 @@
+import atexit
+from queue import Empty,Queue
+from rllab.sampler.utils import rollout
+import numpy as np
```

Comment:
Please format your code according to PEP8. In PEP8, imports are listed alphabetically and sorted into blocks based on their origin.

e.g.
```python
import python.standard.library.dependencies
# empty line
import third.party.dependencies
# empty line
import rllab.dependencies
# empty line
import sandbox.rocky.tf.dependencies
```

Reply:
Done

Reply:
Did you forget to upload your new commits? I don't see them here.

Reply:
Actually, I was under the impression that once I complete the plotter API change then I will add a single commit with all the changes.

Reply:
I need to see all your changes in this pull request. They can be in multiple commits or one squashed commit.